RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptions/ManageSubscriptionsViewModel.swift

```diff
@@ -99,25 +99,24 @@ final class ManageSubscriptionsViewModel: ObservableObject {
 
         switch path.detail {
         case let .feedbackSurvey(feedbackSurvey):
-            self.feedbackSurveyData = FeedbackSurveyData(configuration: feedbackSurvey,
-                                                         path: path) { [weak self] in
-                Task {
-                    await self?.onPathSelected(path: path)
+            self.feedbackSurveyData = FeedbackSurveyData(
+                configuration: feedbackSurvey,
+                path: path) { [weak self] in
+                    Task {
+                        await self?.onPathSelected(path: path)
+                    }
                 }
-            }
+
         case let .promotionalOffer(promotionalOffer):
-            if promotionalOffer.eligible {
```

Comment:
Is the removal of this `if` intentional?

Reply:
nope - reverted it